src/main/groovy/edu/oregonstate/mist/slashcourses/db/InstructorDAO.groovy

```diff
@@ -19,6 +20,29 @@ public interface InstructorDAO extends Closeable{
               """ )
     Instructor getByInstructorID(@Bind("instructorId") Integer instructorId)
 
+    /**
+     * POST a instructor object
+     *
+     * @param lastName
+     * @param firstName
+     */
+    @SqlUpdate("""
+               INSERT INTO INSTRUCTOR (INSTRUCTOR_ID, LAST_NAME, FIRST_NAME)
+               VALUES (INSTRUCTOR_SEQ.NEXTVAL, :lastName, :firstName)
+               """)
+    void postInstructor(@Bind("lastName") String lastName,
```

Comment:
Can the same instructor teach more than one class? The current implementation assumes that whenever a new course is added a new instructor record is added as well.